Release - 1.1.0 (#45)
nf-core/detaxizer: docs/output.md

<details markdown="1">
<summary>Output files</summary>

- `kraken2/`: Contains the output from the kraken2 classification steps.
  - `filtered/`: Contains the classification of the filtered reads (post-filtering).
    - `<sample>.classifiedreads.txt`: The whole kraken2 output for filtered reads.

>     - `<sample>.kraken2.report.txt`: Statistics on how many reads where assigned to which taxon/taxonomic group in the filtered reads.

Reviewer: ```suggestion
    - `<sample>.kraken2.report.txt`: Statistics on how many reads were assigned to which taxon/taxonomic group in the filtered reads.
```

  - `isolated/`: Contains the isolated lines and ids for the taxon/taxa mentioned in the `tax2filter` parameter.
    - `<sample>.classified.txt`: The whole kraken2 output for the taxon/taxa mentioned in the `tax2filter` parameter.
    - `<sample>.ids.txt`: The ids from the whole kraken2 output assigned to the taxon/taxa mentioned in the `tax2filter` parameter.
  - `removed/`: Contains the classification of the removed reads (post-filtering).
    - `<sample>.classifiedreads.txt`: The whole kraken2 output for removed reads.

>     - `<sample>.kraken2.report.txt`: Statistics on how many reads where assigned to which taxon/taxonomic group in the removed reads.

Reviewer: ```suggestion
    - `<sample>.kraken2.report.txt`: Statistics on how many reads were assigned to which taxon/taxonomic group in the removed reads.
```

</details>

### bbduk

> bbduk classifies the reads. The important files are `*.bbduk.log` and `ids/*.bbduk.txt`. `<sample>` can be replaced by `<sample>_longReads`, `<sample>_R1` or left as `<sample>` depending on the cases mentioned in [fastp](#fastp).

Reviewer: ```suggestion
bbduk classifies the reads. The important files are `*.bbduk.log` and `ids/*.bbduk.txt`. `<sample>` can be replaced by `<sample>_longReads`, `<sample>_R1` or left as `<sample>` depending on the cases mentioned in [fastp](#fastp).
```

I find `bbduk classifies the reads` a bit vague... against what exactly and how?

Reviewer: Also as general advice that I learnt from Phil - always put one sentence on a new line (markdown stitches them together unless there is an empty new line), as this makes diffs smaller and easier to see changes, e.g.:

```suggestion
bbduk classifies the reads.
The important files are `*.bbduk.log` and `ids/*.bbduk.txt`.
`<sample>` can be replaced by `<sample>_longReads`, `<sample>_R1` or left as `<sample>` depending on the cases mentioned in [fastp](#fastp).
```

<details markdown="1">
<summary>Output files</summary>

- `bbduk/`: Contains the output from the bbduk classification step.
  - `ids/`: Contains the files with the IDs classified by bbduk.
    - `<sample>.bbduk.txt`: Contains the classified IDs per sample.
  - `<sample>.bbduk.log`: Contains statistics on the bbduk run.

</details>

### classification

> Either the merged IDs from [bbduk](#bbduk) and [kraken2](#kraken2) or the ones produced by one of the tools are shown in this folder. Also, the summary files of the classification step are shown.

Reviewer: ```suggestion
Either the merged IDs from [bbduk](#bbduk) and [kraken2](#kraken2) or the ones produced by one of the tools are shown in this folder. Also, the summary files of the classification step are included.
```

Results generated by MultiQC collate pipeline QC from supported tools e.g. FastQC. The pipeline has special steps which also allow the software versions to be reported in the MultiQC output for future traceability. For more information about how to use MultiQC reports, see <http://multiqc.info>.

### Downstream samplesheets

The pipeline can also generate input files for the following downstream
pipelines:

> - [nf-core/taxprofiler](https://nf-co.re/taxprofiler)

Reviewer: ```suggestion
- [nf-core/taxprofiler](https://nf-co.re/taxprofiler)
- [nf-core/mag](https://nf-co.re/mag)
```

<details markdown="1">
<summary>Output files</summary>

- `downstream_samplesheets/`

>   - `taxprofiler.csv`: Filled out nf-core/taxprofiler `--input` csv with paths to reads relative to the results directory
>   - `mag-pe.csv`: Filled out nf-core/mag `--input` csv for paired-end reads with paths to reads relative to the results directory
>   - `mag-se.csv`: Filled out nf-core/mag `--input` csv for single-end reads with paths to reads relative to the results directory

Reviewer: ```suggestion
  - `taxprofiler.csv`: Filled out nf-core/taxprofiler `--input` csv with paths to reads saved in the results directory
  - `mag-pe.csv`: Filled out nf-core/mag `--input` csv for paired-end reads with paths to reads saved in the results directory
  - `mag-se.csv`: Filled out nf-core/mag `--input` csv for single-end reads with paths to reads saved in the results directory
```

</details>

:::warning
Any generated downstream samplesheet is provided as 'best effort' and are not guaranteed to work straight out of the box!
They may not be complete (e.g. some columns may need to be manually filled in).
:::

:::warning

> Detaxizer can process long-reads independent from short reads. `nf-core/mag@v3.1.0` can only take short, or short+long but not standalone long-reads as an input (this is being worked on). If you want to use the output of Detaxizer for Mag, you'll have to remove the standalone long reads from the `mag.csv` file.

Reviewer: ```suggestion
Detaxizer can process long-reads independent from short reads. nf-core/mag (as of 3.1.0) can only take short, or short + long but not standalone long-reads as an input (this is being worked on). Standalone long-reads will not be included in the nf-core/mag samplesheets.
```

Reviewer: Right, @Joon-Klaps ? We just removed them in the end I think?
